mmv1/products/compute/ForwardingRule.yaml

```diff
@@ -492,7 +492,7 @@ properties:
       For Private Service Connect forwarding rules that forward traffic to managed services, the target must be a service attachment.
     update_url: 'projects/{{project}}/regions/{{region}}/forwardingRules/{{name}}/setTarget'
     update_verb: 'POST'
-    diff_suppress_func: 'tpgresource.CompareSelfLinkRelativePaths'
+    diff_suppress_func: 'tpgresource.CompareSelfLinkRelativePathsWithProjectNumSuppress'
```

Comment:
>           # google_compute_forwarding_rule.forwarding_rule1_network1 will be updated in-place
          ~ resource "google_compute_forwarding_rule" "forwarding_rule1_network1" {
                id                      = "projects/ci-test-project-188019/regions/us-central1/forwardingRules/forwardingrule11"
                name                    = "forwardingrule11"
              ~ target                  = "https://www.googleapis.com/compute/beta/projects/ma3f380586e1b5da2p-tp/regions/us-central1/serviceAttachments/gcp-memorystore-auto-3a9ed5c5-0e48-49-psc-sa" -> "https://www.googleapis.com/compute/v1/projects/91977256640/regions/us-central1/serviceAttachments/gcp-memorystore-auto-3a9ed5c5-0e48-49-psc-sa"
                # (28 unchanged attributes hidden)
        
                # (1 unchanged block hidden)
            }
It looks like this diff suppress isn't working fully, are you getting different results locally?

Reply:
Fixed the issue. It should work now. But facing this new issue https://paste.googleplex.com/5853800871755776